src/python/pants/backend/python/target_types.py

```diff
@@ -238,7 +240,8 @@ def compute_value(
 class PexZipSafeField(BoolField):
     """Whether or not this binary is safe to run in compacted (zip-file) form.
 
-    If they are not zip safe, they will be written to disk prior to execution. iff
+    If they are not zip safe, they will be written to disk prior to execution. You may need to mark
```

Comment:
`they` sounds a little clunky here without some prior use of the antecedent `the files`. Maybe refer to `the binary`, `the files`, or `it`?